src/main/kotlin/com/code_intelligence/cifuzz/android/AndroidTestSetAccess.kt

```diff
@@ -48,4 +48,7 @@ class AndroidTestSetAccess(
 
     override val testTask: Provider<out Task>
         get() = project.provider { project.tasks.named("${testTaskPrefix()}${testComponent().name.capitalized()}") }.flatMap { it }
+
+    override val testSources: FileCollection
+        get() = TODO("Not yet implemented")
```

Comment:
```suggestion
        get() = project.objects.fileCollection().from(testComponent().sources.java?.all, testComponent().sources.kotlin?.all)
```

...and possibly add a corresponding test to `MultiProjectAndroidTest`:

```
        val result = runner("cifuzzPrintTestSourceSets", "-q").build()

        assertEquals(TaskOutcome.SUCCESS, result.task(":app:cifuzzPrintTestSourceSets")?.outcome)
        assertThat(result.output, containsString(File("src/test${testedAndroidVariant().capitalized()}/").path))
        assertThat(result.output, containsString("cifuzz.src-sets.test="))
```

Reply:
great thanks

Reply:
great thanks